go.mod

```diff
@@ -9,36 +9,37 @@ replace (
 
 require (
 	dario.cat/mergo v1.0.0
-	github.com/alexliesenfeld/health v0.6.0
-	github.com/cometbft/cometbft v0.38.6
+	github.com/alexliesenfeld/health v0.8.0
+	github.com/cometbft/cometbft v0.38.7
 	github.com/dgraph-io/badger/v3 v3.2103.5
-	github.com/ethereum/go-ethereum v1.13.8
-	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.0.0-rc.5
-	github.com/grpc-ecosystem/grpc-gateway/v2 v2.18.0
-	github.com/jackc/pglogrepl v0.0.0-20231111135425-1627ab1b5780
-	github.com/jackc/pgx/v5 v5.5.2
+	github.com/ethereum/go-ethereum v1.14.3
+	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.1.0
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.19.1
+	github.com/jackc/pglogrepl v0.0.0-20240307033717-828fbfe908e9
+	github.com/jackc/pgx/v5 v5.5.5
 	github.com/jpillora/backoff v1.0.0
-	github.com/kwilteam/kwil-db/core v0.1.2
-	github.com/kwilteam/kwil-db/parse v0.0.1
+	github.com/kwilteam/kwil-db/core v0.2.0-beta
+	github.com/kwilteam/kwil-db/parse v0.2.0-beta
```

Comment:
Obviously these tags don't actually exist.  It's more of documentation, with the `replace`s making it work for now.  We can probably make `beta.1` tags on Monday for these modules though.